src/compat/dbtype.h

```diff
@@ -791,6 +791,9 @@ union db_char
     unsigned char codeset;
     int size;
     char *buf;
+    int compressed_length;
```

Comment:
Please rename it as compressed_size, since length has the meaning of number of characters.